genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -976,4 +976,24 @@ The server also has a chance to override them during the 'Agent Configuration' e
 | 1.2
 |
 
+| `genie.agent.runtime.shutdown.internal-executors-leeway`
```

Comment:
There are enough agent properties at this point that we might want to think about breaking them out into two separate docs for agent and server properties